game/hud/src/widgets/Social/components/OrdersGrid.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * @Author: Mehuge (mehuge@sorcerer.co.uk)
+ * @Date: 2017-03-08
+ */
+
+import * as React from 'react';
+import { graphql, InjectedGraphQLProps } from 'react-apollo';
+import { connect } from 'react-redux';
+import gql from 'graphql-tag';
+import { StyleSheet, css, StyleDeclaration } from 'aphrodite';
+import { Spinner, ColumnDefinition } from 'camelot-unchained';
+import GridViewPager from './GridViewPager';
+import GroupTitle from './GroupTitle';
+
+export interface OrdersGridStyle extends StyleDeclaration {
+  container : React.CSSProperties;
+}
+
+export const defaultOrdersGridStyle : OrdersGridStyle = {
+  container: {
+    flex: '1 1 auto',
+    display: 'flex',
+    flexDirection: 'column'
+  },
+};
+
+export interface OrdersGridQuery {
+  orders: {
+    totalCount: number;
+    data: {
+      id: string;
+      name: string;
+    }[]
```

Comment:
This data type is wrong.  data has the properties id, name, realm, creator, created.  This also makes me think it would be good to have member count as well, so I'll make a note to add that as a field you can get on an order from graphql without having to pull all the members down.